src/platform/utilities/tests/sso.unit.spec.js

```diff
@@ -21,6 +26,74 @@ function setKeepAliveResponse(stub, sessionTimeout = 0) {
 
 const SSO_SESSION_TIMEOUT = 900; // seconds
 
+describe('checkAutoSession', () => {
+  it('should auto logout if user is logged in and they do not have a SSOe session', async () => {
+    const sandbox = sinon.createSandbox();
+    sandbox.stub(profUtils, 'hasSession').returns(true);
+    sandbox.stub(profUtils, 'hasSessionSSO').returns(false);
+    sandbox.stub(forceAuth, 'getForceAuth').returns(undefined);
```

Comment:
```suggestion
    sandbox.stub(profUtils, 'hasSession').returns('true');
    sandbox.stub(profUtils, 'hasSessionSSO').returns(false);
    sandbox.stub(forceAuth, 'getForceAuth').returns(null);
```

`localStorage` returns strings from getting items. I suppose the behavior could be more consistent if `hasSession` also had its value `JSON.parse`'d.

And I believe `localStorage` returns `null` for non-existent items.

Reply:
yeah just tested that out, it does return `null`